src/tools/index.md

```diff
@@ -103,23 +102,12 @@ The Dart SDK includes the following general-purpose `dart` tool:
 : A suite of debugging and performance tools.
 
 
-## Tools for developing web apps {#web}
+## Tool for developing web apps {#web}
 
 The following tools support developing web apps:
 
 [`webdev`](/tools/webdev)
-: A CLI for Dart web app development,
-  including building and serving web apps.
-
-[dart2js](/tools/dart2js)
-: The original Dart-to-JavaScript compiler, with tree shaking.
-  IDEs, `dart compile js`, and the `webdev` CLI 
-  use dart2js when building web apps for deployment.
-
-[dartdevc](/tools/dartdevc)
-: The Dart dev compiler, a modular Dart-to-JavaScript compiler.
-  IDEs and the `webdev` CLI use dartdevc when running a development server.
-
+: A CLI to build and serve Dart web apps.
```

Comment:
Should we consider adding back a bullet point equivalent to `dart2js` (maybe it would be `dart compile js` instead?).

It seems these were the primary links we had to steer readers towards the more detailed articles about how to develop and deploy apps to JavaScript, so without those, this index page feels incomplete.

Also worth nothing something for a follow up (clearly out of scope for this change): it's strange that have a couple workflow articles like web/deployment and web/debugging, but no article on web/development. It would be be consistent and have an article per user journey. My guess the reason is that it overlaps entirely with the content of the webdev.md article, so it would require a bit of editing to figure out the right organization of that content.